DuplicateAssemblyScanner/DuplicateAssemblyScanner/DuplicateAssemblyScanner.csproj

```diff
@@ -42,33 +42,51 @@
     <MangedDLLPath Condition="! Exists ('$(MangedDLLPath)')">..\dependencies</MangedDLLPath>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Assembly-CSharp">
+    <Reference Include="Assembly-CSharp, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null">
```

Comment:
Why would you think this is important here?

Reply:
Following the approach taken in https://github.com/CitiesSkylinesMods/PathfindSandbox/blob/master/PathfindSandbox/PathfindSandbox.csproj

Reply:
You only need to specify a specific version if there is a requirement to do so. (E.g. your code depends on a feature that is only available in that specific version, or you know a major bug in other versions so you want to avoid it). In all other cases, just use the assembly name, without a version.
Specifying assembly versions and key tokes for the system references (like `mscorlib`) is meaningless, because there is only *one* version of that assembly in a given runtime anyway (including Unity/Mono). You have no option to get a different one.
Requesting version `0.0.0.0` is quite awkward too (that is because Unity generates that dummy version stub on building the game assembly).

Reply:
Will sort this out in separate project, I'm splitting out a load of common props from the csproj file so they can be shared across multiple projects/repos.